hms_tz/nhif/doctype/nhif_patient_claim/nhif_patient_claim.js

```diff
@@ -34,12 +34,16 @@ frappe.ui.form.on('NHIF Patient Claim', {
 			});
 
 			frm.add_custom_button(__("Merge Claims"), function () {
+				const original_practitioner_name = frm.doc.practitioner_name ;
+				const original_practitioner_no = frm.doc.practitioner_name;
 				frm.dirty()
 				frm.call('get_appointments', {
 					self: frm.doc,
 					freeze: true,
                 	freeze_message: __('<i class="fa fa-spinner fa-spin fa-4x"></i>'),
 				}).then(r => {
+					frm.doc.practitioner_name = original_practitioner_name;
+                  	frm.doc.practitioner_no = original_practitioner_no;
```

Comment:
overriding practitioner no, is a problem, do not override 'practitioner_no' because if you merge practitioner_no should have more than one mct code, but practitioner name can remain same

please make change here